espresso/environment/3_2_espresso_deterministic_state_test.go

```diff
@@ -105,24 +124,251 @@ func TestDeterministicDerivationExecutionState(t *testing.T) {
 			t.Fatalf("Waiting for L2 tx:\nhave:\n\t\"%v\"\nwant:\n\t\"%v\"\n", have, want)
 		}
 
-		// Get latest safe blocks from caff node first
-		// as caff node usually lags behind the sequencer node on safe blocks due to submitting additionally to Espresso.
-		// We use l2BlockRefByLabel to get the states as the engine state will be reflected in the block.
-		caffBlock, err := caffNodeL2Client.L2BlockRefByLabel(ctx, eth.Safe)
+		// When it is the attack round, try to send some Espresso transactions with fakeBatcherPrivateKey directly to Espresso, outside of the batcher.
```

Comment:
```suggestion
		// When it is the attack round, send some Espresso transactions with fakeBatcherPrivateKey directly to Espresso.
```

Reply:
No need to mention "outside" the batcher as the batcher runs in a TEE and thus it would not be possible to use a malicious while running the batcher's code.